<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: none;
background-color: rgb(30, 30, 30);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>560</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(150, 150, 150);
border-style: none;
border-radius: 24px;</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>560</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(150, 150, 150);
border-style: none;
border-radius: 24px;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhEmailCharactersOnly</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>391</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(76, 76, 76);
background-color: rgb(75, 75, 75);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="createNewAccount">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>601</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(150, 150, 150);
border-style: none;
border-radius: 24px;</string>
    </property>
    <property name="text">
     <string>Don't have an account  yet? Create one</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>320</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 1px;
border-radius: 18px;
border-color: rgb(150, 150, 150);
color: rgb(150, 150, 150);</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/right-arrow (1).png</normaloff>icons/right-arrow (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
